packages/blockchain-wallet-v4-frontend/src/data/components/buySell/sagas.ts

```diff
@@ -2217,15 +2243,25 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
     }
   }
```

Comment:
Yeah totally. The issue is that since this call (and the products call) is in the saga, it makes a few seconds delay from when you click the button to when the flyout flies out. I think what we could do though is `dispatch()` these api calls from a `useEffect` in the component and set the response as a remote type in redux. So then the flyout will fly out immediately after clicking the button, but will sit on the loading state until all the api calls resolve.